src/main/java/com/k3ntako/HTTPServer/controllers/Reminders.java

```diff
@@ -36,13 +36,14 @@ public Response get(RequestInterface request) throws IOException, HTTPError {
     var params = request.getParams();
 
     var id = params.get("id");
-    if (id == null || id.isBlank()) {
-      throw new HTTPError(400, "ID must be specified");
-    }
 
     var response = new Response();
 
     var content = textFile.readFile(id);
+    if (content == null) {
+      throw new HTTPError(404, "Reminder was not found");
```

Comment:
just a thought here - is it possible to create a file/reminder with no content?

Reply:
An empty file will return an empty string, so no issues here. But, as you suggested I added a test to make sure and document it.